### DISCUSSION TITLE
Can get Create3 Topics on RPi connected via USB, but not on Dev machine (Dell) via WiFi

### DISCUSSION THREAD

**Discussion by ldbrouwer** [iRobotEducation/create3_docs#589] [Questions] [answered]
### How are you connecting to your Create 3?

Wi-Fi and USB Ethernet (ROS 2)

### Computer(s) Model(s) and Operating System(s)

Rasp Pi 4 with Ubuntu 22.04 and Dell Inspiron with Ubuntu 22.04

### Which version of ROS 2 is installed on your computer?

Humble

### Which firmware version is installed on your robot?

H2.6

### Which RMW is your robot running?

FastRTPS

### Does your robot have an assigned namespace? If so, please share.

none

### Is the robot connected to a network? If so, what is the network type?

yes, home network

### Are there multiple Create 3 robots connected to your network?

No

### Is multicast enabled?

None

### What is the Adapter Board's USB/BLE Toggle currently switched to?

USB

### Describe your question.

I followed the Create3 setup instructions for both the RPi and the Dell. I can use the RPi with the Create3 and have written several Nodes that I can use with the RPI. However, I have never been able to connect my Dell via wifi and see the ROS2 topics. I have set the environment for fast dds and can verify it's use via "echo $RMW_IMPLEMENTATION" on the RPi and Dell. I also can verify it on the Create3 webpage. Do I need to use the Discovery Server? Lonnie

**Comment by alsora** [comment, marked as answer]
Are you sure that the Create 3, the raspberry Pi and the laptop are all connected to the same wifi network, i.e. your home network?
On your laptop, what's the output of this command? 
```
ros2 topic list --spin-time 10 --no-daemon
```
do you see something with it? (you should see both the create 3 and the RPi topics).

In general, are you able to communicate via ROS between the laptop and the Pi?

**Comment by ldbrouwer** [comment]
Alberto,
All of the computers are on the same router and able to communicate, but the Dell was on the 5G radio and the Create3 and RPi were on the 2.4G radio. When I switched the Dell to the 2.4G, I could see the topics. Obviously, the bridge in the router doesn't help with the RMW - it needs to be on the same radio (or SSID). I just hadn't thought about that since the computers could talk together. Problem solved! Thank you.
